stable/insights-agent/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 0.7.6
+version: 0.8.0
```

Comment:
New to how this version works. Reason to skip to `0.8.0` ?

Reply:
Since the major version of the goldilocks chart has gone from 1-2, I felt that this would be a potential significant change in insights and bumped the minor version instead of patch.